tools/nni_trial_tool/trial_runner.py

```diff
@@ -27,6 +27,10 @@ def main_loop(args):
     gpu_refresh_last_time = datetime.now() - timedelta(minutes=1)
 
     try:
+        if args.job_pid_file:
```

Comment:
If use same way like openpai, it doesn't need here.

Reply:
This pid file is used to check trial_runner process status for environment